pkg/objectstore/azure/azure.go

```diff
@@ -24,14 +30,46 @@ func getPipeline(backupLocation *stork_api.BackupLocation) (pipeline.Pipeline, e
 	return azureblob.NewPipeline(credential, azblob.PipelineOptions{}), nil
 }
 
+func getAzureURLSuffix(backupLocation *stork_api.BackupLocation) (string, error) {
+	// Give first preference to environment variable setting.
+	azureEnv := os.Getenv(azureEnvKey)
+	if len(azureEnv) == 0 {
+		// If env variable is not set, check the azure environment vaue from backuplocation CR.
+		if len(backupLocation.Location.AzureConfig.Environment) != 0 {
+			azureEnv = backupLocation.Location.AzureConfig.Environment
+		}
```

Comment:
Maybe we should have a log messgae which says where the Endpoint is picked up from. Either Env or BL CR, useful for debugging

Reply:
Added the log message